packages/manager/src/features/CloudPulse/shared/CloudPulseRegionSelect.tsx

```diff
@@ -50,8 +50,10 @@ export const CloudPulseRegionSelect = React.memo(
         data-testid="region-select"
         disableClearable={false}
         disabled={!selectedDashboard || !regions}
+        errorText={isError ? `Failed to fetch ${label || 'Regions'}` : ''}
```

Comment:
If label itself is undefined?

Reply:
@kmuddapo , if you see it is handled if label is undefined, we will show 
**Failed to fetch Regions**